trunk/NDHMS/Land_models/NoahMP/IO_code/module_NoahMP_hrldas_driver.F

```diff
@@ -1518,7 +1523,7 @@ subroutine land_driver_ini(NTIME_out,wrfits,wrfite,wrfjts,wrfjte)
        print*, "t0OutputFlag: ", t0OutputFlag  
 #endif        
 !#ifdef HYDRO_REALTIME
-   if(t0OutputFlag .eq. 1) call ldas_output()
+   if(t0OutputFlag .eq. 1) call ldas_output(0)
```

Comment:
I love how this is so opaque. What was the assumed value when nothing was passed, what does 0 represent. Is there no good variable that can be used?

Reply:
hmm, that is the initialization timestep, actually I could use ITIME as I used it in the other routines. I will check why I did not use it, and it possible I would replace it with ITIME.